pkg/util/testingjobs/rayjob/wrappers.go

```diff
@@ -151,6 +151,11 @@ func (j *JobWrapper) WithWorkerPriorityClassName(value string) *JobWrapper {
 	return j
 }
 
+func (j *JobWrapper) WithNumOfHosts(value int32) *JobWrapper {
+	j.Spec.RayClusterSpec.WorkerGroupSpecs[0].NumOfHosts = value
+	return j
+}
+
```

Comment:
Maybe it's better to put it directly to `rayv1.WorkerGroupSpec`?`

Reply:
Ah, good catch